locales/sq_AL/LC_MESSAGES/duckduckgo.po

```diff
@@ -474,69 +476,69 @@ msgstr "Bazë"
 
 msgctxt "theme"
 msgid "Basic"
-msgstr ""
+msgstr "Tema"
 
 msgctxt "precise_user_location"
 msgid "Before you go, please help us understand why:"
-msgstr ""
+msgstr "Para se të na lini, ju lutem na ndihmoni të kuptojmë pse"
 
 msgid "Belgium (fr)"
-msgstr ""
+msgstr "Belgjikë"
```

Comment:
This and the string at line 489 differ because of the specifier `(fr)` here and `(nl)` there. They specify specific variants of the language (French Belgian and Dutch Belgian), please keep the specifiers.
```suggestion
msgstr "Belgjikë (fr)"
```